config/locales/api.en.yml

```diff
@@ -25,7 +62,16 @@ en:
           format: int
           desc: "Total number of existing records"
 
+      request: |-
+        GET /api/v1/plant_lines?page=2&per_page=100 HTTP/1.1
+        Content-Type: application/json
+        Accept: application/json
+        X-BIP-Api-Key: ba6d4625927ede0db8768782994d9276cfff9ba79e8df8c831aedc453e850836
```

Comment:
I think it's all right as it is, with GET APIs. Just wanted to know what was the rationale.